upstairs/src/test.rs

```diff
@@ -230,10 +231,12 @@ mod up_test {
     }
 
     #[tokio::test]
-    #[should_panic]
```

Comment:
I think we should put back the assertions if invalid input is given to `extent_from_offset`,
however, if we don't, then maybe change this test to "off_to_extent_ignore_length_to_big"?